README.md

```diff
@@ -17,7 +13,7 @@ cache.
 
 *   Launch `bazels3cache` like this (by default it listens on port 7777):
 
-        bazels3cache --bucket=MY_S3_BUCKET
```

Comment:
is `MY_S3_BUCKET` user specific, or should we all be pointing to the same aws bucket?

Reply:
@balderdash ^

Reply:
We'll all use the same bucket. This is a public fork of bazels3cache so I'm leaving the placeholder arguments here. I'll put the specific arguments you'll want to use in the audere mediapipe README.

Reply:
ahh got it. Sorry I though I was looking at the `mediapipe` repo